schemas/inst_schema.json

```diff
@@ -182,6 +180,19 @@
           "type": "string",
           "description": "Short description of the instruction"
         },
+        "type": {
+          "type": "string",
+          "enum": [
+            "I-Type",
+            "S-Type",
+            "B-Type",
+            "U-Type",
+            "J-Type",
+            "R-Type",
```

Comment:
Let's lowercase the "-type" suffix, as is done in the spec when not used in a title.